### ISSUE TITLE
Plots Losing Owners

### ISSUE THREAD

**Issue by Aberzhulan** [IntellectualSites/PlotSquared#600]
A large number (over half, by last count) of plots on my Creative server are losing owners after server reboot (plot owners set to unknown). I've turned on sqluuidhandler, have not deleted any playerdata files, and this is not a plotme conversion so UUID cache is off. I took a peek into the storage.db and have noticed some plots have UUIDs that don't exist, according to Mojang. A few of those plots, however, are set to the correct owners. Also, I've noticed almost every single plot in the database is listed twice, but with different IDs. It appears every time we set owner it adds another line to the database.

**Comment by EvilOlaf** [comment]
Do you use the latest available version of P^2. A similar issue has been fixed already: https://github.com/IntellectualSites/PlotSquared/issues/550

**Comment by Aberzhulan** [comment]
I do. We had somewhat unrelated issues on 8/30 that caused our server to be down for about a day. On 8/29 I claimed 4 plots and today they were showing as unknown. I actually had to look in the database to figure out where my plots were. Which is something I find super unusual. The database lists 11 plots for me, but in game it shows 4.

**Comment by boy0001** [comment]
Offline mode or online mode?
Are you able to provide a comparison of the following uuids:
 - Your mojang uuid.
 - UUID of /plot debugexec seen <player>
 - UUID in the database
 - Your username

**Comment by boy0001** [comment]
Another thing, if you aren't on the latest version, please update.

**Comment by Aberzhulan** [comment]
Online mode. We're using a BungeeCord, if it matters.

Mojang UUID: 9de113d2539b462a9343c3fdd18afe75
UUID of /plot debugexec seen: 9de113d2-539b-462a-9343-c3fdd18afe75
UUID in database: 9de113d2-539b-462a-9343-c3fdd18afe75
Username: Aberzhulan

It all looks the same, except for those dashes, but I wouldn't think that would matter much. Also, we are on the latest version.

**Comment by Aberzhulan** [comment]
So it turns out the MySQL database generated by PlotSquared didn't have a unique ID for each row, and that may have been causing the error. After we fixed that, set the plot owners, and then rebooted the server all plots are owned by the correct players.

Also, in original post, the database I was referring to is the SQLite database. We switched after to see if it was an issue with the SQLite database.
